fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePartitioner.java

```diff
@@ -92,6 +94,13 @@
  */
 @API(API.Status.EXPERIMENTAL)
 public class LucenePartitioner {
+    /**
+     * partition error reasons.
+     */
+    public enum PartitionFailureReason {
```

Comment:
TODO evaluate if still needed. This was for when there were potentially multiple situations to report/handle.